### PR TITLE
feat: upgrade to Chromium 69.0.3497.106

### PR DESCRIPTION
**Pull request by alexeykuzmin**
##### Description of Change

Upgrade to Chromium 69.0.3497.106
BREAKING CHANGE

Board https://github.com/orgs/electron/projects/24

##### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

##### Release Notes

Notes: Upgrade to Chromium 69.0.3497.106

### CODE REVIEW

**Comment by KoenLav** [comment]
The speed you guys (Electron) have taken up in keeping up with Chromium versions is incredible!

Do you know which version of Chromium is expected to land in 4.0?

**Comment by nornagon** [comment]
> Do you know which version of Chromium is expected to land in 4.0?

Not yet. We have a go/no-go for M69 on Monday. It'll definitely be either M68 or M69. We'll know soon!

File: BUILD.gn [hunk 1]
```diff
@@ -1,958 +1,1024 @@
...
-
-service_manifest("electron_content_packaged_services_manifest_overlay") {
-  source = "//electron/manifests/electron_content_packaged_services_manifest_overlay.json"
-  packaged_services = [ "//services/proxy_resolver:proxy_resolver_manifest" ]
-}
-
-service_manifest("electron_content_browser_manifest_overlay") {
-  source = "//electron/manifests/electron_content_browser_manifest_overlay.json"
-}
+import("build/asar.gni")
```
Comment (MarshallOfSound) [inline BUILD.gn]: @brenca I think you committed the CLRF line endings for this file instead of just LF

File: BUILD.gn [hunk 2]
```diff
@@ -485,12 +485,12 @@ static_library("electron_lib") {
...
       "//chrome/renderer/prerender/prerenderer_client.cc",
       "//chrome/renderer/prerender/prerenderer_client.h",
-      "//chrome/utility/printing_handler.cc",
-      "//chrome/utility/printing_handler.h",
     ]
 
     if (is_win) {
       sources += [
+        "//chrome/utility/printing_handler.cc",
+        "//chrome/utility/printing_handler.h",
```
Comment (alexeykuzmin) [inline BUILD.gn]: @brenca `sources` should be sorted alphabetically, like any other lists in GN configs.
The linter job didn't report its results for some reason.

Comment (brenca) [reply BUILD.gn]: I had to fix the linter, it refused to run for me on Windows, thanks for catching this!

**Comment by KoenLav** [comment]
@nornagon what was decided for 4.0?

**Comment by ckerr** [comment]
@KoenLav 4.0.0 will have Chromium 69

**Pull request merged**